roles/base/tasks/main.yml

```diff
@@ -54,16 +76,24 @@
   copy:
     dest: /etc/profile.d/00golang.sh
     content: "export PATH=/opt/golang/bin:/usr/local/go/bin:$PATH\nexport GOPATH=/opt/golang"
+- name: remove older docker
```

Comment:
sure, I was thinking to rewrite the `docker install` task as follows. This is just a sample to show the idea and might have bugs. We can also wrap all of this is in a single docker install script as well and replace multiple tasks with one. The benefit is that say in future if docker really start providing a way to pull specific versions, we can just replace our custom script with the docker's script:

```
- stat: path=/usr/bin/docker
  register: docker_bin

- shell:  docker version | awk '/Version/{print $2; exit 0}' 
  register: docker_version

- name: install docker
  shell: curl https://get.docker.com | bash
  when: docker_bin.exists == False or docker_ver.output < 1.9
```